File: AMS_资讯信息组件/code/client/source/YssSecInformation.Support/Mp/SecEq/Pojo/TdChan.cs
using FAST.Core.Util;
using FAST.Common.Service.Pojo;
using FAST.Common.Service.Pojo.Base;
using FAST.Core.Exceptions;


using FAST.Core.Context;

using Newtonsoft.Json;
using FAST.Common.Service.Datastructure;

namespace YssSecInformation.Support.Mp.SecEq.Pojo
{
    /// <summary>
    /// /// /// chenbo
    /// 2017-06-22
    /// #42948 资讯信息管理组件化拆分
    /// ----------------------------------
    /// 证券交易渠道
    /// </summary>
    [NodeDesc(ParentNode = "C_DV_CHAN_TYPE", TreeNode = "C_TD_CHAN_CODE")]
    public class TdChan : AuditableParamPojo
    {
        /// <summary>
        ///   交易渠道代码
        /// </summary>
        private string c_TD_CHAN_CODE = "";

        /// <summary>
        ///  交易渠道名称
        /// </summary>
        private string c_TD_CHAN_NAME = "";

        /// <summary>
        /// 机构代码
        /// </summary>
        private string c_ORG_CODE = "";

        /// <summary>
        /// 交易所代码
        /// </summary>
        private string c_MKT_CODE = "";

        /// <summary>
        /// 渠道类型
        /// </summary>
        private string c_DV_CHAN_TYPE = "";

        /// <summary>
        /// 描述
        /// </summary>
        private string c_DESC = "";

        /// <summary>
        /// 属性: 交易渠道代码 
        /// </summary>
        [JsonProperty(PropertyName = "c_TD_CHAN_CODE")]
        public string C_TD_CHAN_CODE
        {
            set { c_TD_CHAN_CODE = value; }

            get { return c_TD_CHAN_CODE; }
        }

        /// <summary>
        /// 属性: 交易渠道名称 
        /// </summary>
        [JsonProperty(PropertyName = "c_TD_CHAN_NAME")]
        public string C_TD_CHAN_NAME
        {
            set { c_TD_CHAN_NAME = value; }

            get { return c_TD_CHAN_NAME; }
        }

        /// <summary>
        /// 属性: 机构代码 
        /// </summary>
        [JsonProperty(PropertyName = "c_ORG_CODE")]
        public string C_ORG_CODE
        {
            set { c_ORG_CODE = value; }

            get { return c_ORG_CODE; }
        }

        /// <summary>
        /// 属性: 交易所代码 
        /// </summary>
        [JsonProperty(PropertyName = "c_MKT_CODE")]
        public string C_MKT_CODE
        {
            set { c_MKT_CODE = value; }

            get { return c_MKT_CODE; }
        }

        /// <summary>
        /// 属性: 渠道类型 
        /// </summary>
        [JsonProperty(PropertyName = "c_DV_CHAN_TYPE")]
        public string C_DV_CHAN_TYPE
        {
            set { c_DV_CHAN_TYPE = value; }

            get { return c_DV_CHAN_TYPE; }
        }

        /// <summary>
        /// 属性: 描述 
        /// </summary>
        [JsonProperty(PropertyName = "c_DESC")]
        public string C_DESC
        {
            set { c_DESC = value; }

            get { return c_DESC; }
        }


    }
}

File: AMS_资讯信息组件/code/server/source/com.yss.ams.sec.information.support/src/main/java/com/yss/ams/sec/information/support/modules/mp/secTransfer/pojo/TdChan.java
package com.yss.ams.sec.information.support.modules.mp.secTransfer.pojo;

import com.yss.framework.api.common.co.AuditableParamPojo;

/**
 * #42948 资讯信息管理组件化拆分
 * @author chenbo
 *2017-07-06
 */
public class TdChan extends AuditableParamPojo {
	/**
	* 交易渠道代码 
	*/
	  private String c_TD_CHAN_CODE = "";

	/**
	* 交易渠道名称 
	*/
	  private String c_TD_CHAN_NAME = "";

	/**
	* 机构代码 
	*/
	  private String c_ORG_CODE = "";

	/**
	* 交易所代码 
	*/
	  private String c_MKT_CODE = "";

	/**
	* 渠道类型 
	*/
	  private String c_DV_CHAN_TYPE = "";

	/**
	* 描述 
	*/
	  private String c_DESC = "";

	private static final long serialVersionUID = 1L ;

	public String getC_TD_CHAN_CODE() {
		return c_TD_CHAN_CODE;
	}

	public void setC_TD_CHAN_CODE(String cTDCHANCODE) {
		c_TD_CHAN_CODE = cTDCHANCODE;
	}

	public String getC_TD_CHAN_NAME() {
		return c_TD_CHAN_NAME;
	}

	public void setC_TD_CHAN_NAME(String cTDCHANNAME) {
		c_TD_CHAN_NAME = cTDCHANNAME;
	}

	public String getC_ORG_CODE() {
		return c_ORG_CODE;
	}

	public void setC_ORG_CODE(String cORGCODE) {
		c_ORG_CODE = cORGCODE;
	}

	public String getC_MKT_CODE() {
		return c_MKT_CODE;
	}

	public void setC_MKT_CODE(String cMKTCODE) {
		c_MKT_CODE = cMKTCODE;
	}

	public String getC_DV_CHAN_TYPE() {
		return c_DV_CHAN_TYPE;
	}

	public void setC_DV_CHAN_TYPE(String cDVCHANTYPE) {
		c_DV_CHAN_TYPE = cDVCHANTYPE;
	}

	public String getC_DESC() {
		return c_DESC;
	}

	public void setC_DESC(String cDESC) {
		c_DESC = cDESC;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}

}
